tests/atest/rules/comments/missing-space-after-comment/expected_output.txt

```diff
@@ -1 +1,26 @@
-${rules_dir}${\}test.robot:17:18 [W] 0702 Missing blank space after comment character
\ No newline at end of file
+${rules_dir}${\}test.robot:2:20 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:3:20 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:4:21 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:5:21 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:6:25 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:7:25 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:8:25 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:9:25 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:10:25 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:14:16 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:18:6 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:20:21 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:21:22 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:22:25 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:23:19 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:27:0 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:29:18 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:30:9 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:31:34 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:32:25 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:33:19 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:34:18 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:38:16 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:40:29 [W] 0702 Missing blank space after comment character
+${rules_dir}${\}test.robot:42:10 [W] 0702 Missing blank space after comment character
```

Comment:
I don't know why it discovers everything from RF4 execpt for `test.robot:45:30 [W] 0702 Missing blank space after comment character` which is:
```
    FOR  ${var}  IN  @{list}  #invalid comment
```

Reply:
Isn't it because you're limiting this check to IS_RF4?
```
    def visit_ForHeader(self, node):  # noqa
        if IS_RF4:
            self.find_comments(node)
            self.generic_visit(node)

```

Reply:
Ok I know why. It's called `ForLoopHeader` in 3.2 (and that's what you should visit for RF 3.2). So you need both visit_ForHeader and visit_ForLoopHeader

Reply:
Added missing visitor and also removed condition for checking RF version. Now it works fine.